modules/nw-operator-cr.adoc

```diff
@@ -112,20 +89,26 @@ allocated and the subnet prefix length assigned to each individual node.
 Interface (CNI) plug-in supports only a single IP address block for the service
 network.
 
-<3> {default-network}
+<3> Configures the Pod network provider for the cluster network.
 
 <4> The parameters for this object specify the Kubernetes network proxy
-(kube-proxy) configuration.
+(kube-proxy) configuration. If you are using the OVN-Kubernetes network
+provider, the kube-proxy configuration has no effect.
+
+<5> The refresh period for `iptables` rules. The default value is `30s`. Valid suffixes include `s`, `m`, and `h` and are described in the link:https://golang.org/pkg/time/#ParseDuration[Go time package] documentation.
 
-<5> {kube-proxy-refresh}
-<6> {iptables-min-sync-period}
+<6> The minimum duration before refreshing `iptables` rules. This parameter ensures that the refresh does not happen too frequently. Valid suffixes include `s`, `m`, and `h` and are described in the link:https://golang.org/pkg/time/#ParseDuration[Go time package].
 endif::operator[]
 
 [id="nw-operator-configuration-parameters-for-openshift-sdn_{context}"]
-== Configuration parameters for OpenShift SDN
+== Configuration parameters for the OpenShift SDN network provider
 
 The following YAML object describes the configuration parameters for
-OpenShift SDN:
+the OpenShift SDN Pod network provider.
+
+ifdef::operator[]
+NOTE: You can only change the configuration for your Pod network provider during cluster installation.
```

Comment:
This conditionalized note is not appearing in preview build: https://osdocs-745--ocpdocs.netlify.com/openshift-enterprise/latest/networking/cluster-network-operator.html#nw-operator-cr_cluster-network-operator 
Please double-check that this is correctly applied (I'm sure it is, but calling out just in case).

Reply:
@bobfuru, that's odd as it is working locally. I wonder if the preview wasn't rebuilt.